kodilla-stream/src/test/java/com/kodilla/stream/portfolio/BoardTestSuite.java

```diff
@@ -135,4 +136,20 @@ public void testAddTaskListFindLongTasks() {
         //Then
         Assert.assertEquals(2, longTasks);
     }
+
+    @Test
+    public void testAddTaskListAverageWorkingOnTask() {
+        //Given
+        Board board = prepareTestData();
+        //When
+        List<TaskList> inProgressTasks = new ArrayList<>();
+        inProgressTasks.add(new TaskList("In progress"));
+        double average = board.getTaskLists().stream()
+                .filter(inProgressTasks::contains)
+                .flatMap(task -> task.getTasks().stream())
+                .mapToDouble(days -> ChronoUnit.DAYS.between(days.getCreated(), LocalDate.now()))
+                .average().getAsDouble();
```

Comment:
tutaj breakline :)
lepiej zamiast `getAsDuble` dać `orElse(0)`, gdyż jeśli nic nie będzie, to zamiast zwrócić zero, rzuci wyjątkiem :)